lib/y2logsstep.pm

```diff
@@ -85,9 +85,7 @@ sub break_dependency {
     if (check_var('VIDEOMODE', 'text')) {
         while (check_screen('dependency-issue-text', 5)) {    # repeat it untill all dependency issues are resolved
             wait_screen_change { send_key 'alt-s' };          # Solution
-            while (check_screen('break-not-highlighted', 5)) {
-                send_key 'down';                              # down to option break dependency
-            }
```

Comment:
wouldn't this break the operation in GUI-mode? Please provide verification runs for both textmode and GUI mode